script/prepack.sh

```diff
@@ -47,7 +48,9 @@ fi
 # copy the config files if can't find the config env
 if [ ! -d "$CONFIG_PATH" ]; then
   mkdir $CONFIG_PATH
+  mkdir -p "${PACK_PATH}/aion/${DEFAULT_NETWORK}/config"
   cp -r ./modBoot/resource/** $CONFIG_PATH
+  cp  ${CONFIG_PATH}/${DEFAULT_NETWORK}/** "${PACK_PATH}/aion/${DEFAULT_NETWORK}/config"
```

Comment:
I don't understand the purpose of this change.

Reply:
cause package the binary need to get the version number. If the cfgAion can't load the fork files. it will print something out. them the package will be mess up.